kernel/solo5.h

```diff
@@ -136,8 +165,13 @@ uint64_t solo5_clock_monotonic(void);
 uint64_t solo5_clock_wall(void);
 
 /*
- * Blocks until monotonic time reaches until_nsecs or I/O is possible,
- * whichever is sooner. Returns 1 if I/O is possible, otherwise 0.
+ * Blocks until monotonic time reaches until_nsecs or network I/O is possible,
+ * whichever is sooner.
+ *
+ * If network I/O is possible:
+ *     Returns 1.
+ * If monotonic time has reached until_nsecs and network I/O is not possible:
+ *     Returns 0.
  *
  * TODO: Extend this interface to select which I/O events are of interest.
  */
```

Comment:
Agree, going for `solo5_wait()` instead of `solo5_poll()` might be the easiest way to avoid confusion.